model/shared/src/main/scala/explore/model/Page.scala

```diff
@@ -18,6 +18,7 @@ object Page {
   final case object ProposalPage         extends Page
   final case object ObservationsBasePage extends Page
   final case class ObsPage(obsId: Observation.Id) extends Page
+  final case class ObsAdvancedConfPage(obsId: Observation.Id) extends Page
```

Comment:
I'm not sure this is the way to proceed.

How are we going to show the sequence editor? Will it be within the Observations View?